dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -56,6 +56,7 @@
 import componenttest.exception.TopologyException;
 import componenttest.logging.ffdc.IgnoredFFDCs;
 import componenttest.logging.ffdc.IgnoredFFDCs.IgnoredFFDC;
+import componenttest.rules.repeater.EE9PackageReplacementHelper;
```

Comment:
I'd suggest updating the copyright date range for this file. Looks like pretty significant functional changes.